### PR TITLE
HDDS-4437. Avoid unnecessary builder conversion in setting volume Quota/Owner request

### PR DESCRIPTION
**Pull request by aeioulisa**
## What changes were proposed in this pull request?
Avoid unnecessary call of toBuilder.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/projects/HDDS/issues/HDDS-4437

## How was this patch tested?
Using the existed UTs

(Please explain how this patch was tested. Ex: unit tests, manual tests)
(If this patch involves UI changes, please attach a screen-shot; otherwise, remove this)

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetOwnerRequest.java [hunk 1]
```diff
@@ -65,12 +65,12 @@ public OMVolumeSetOwnerRequest(OMRequest omRequest) {
   public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
 
     long modificationTime = Time.now();
-    SetVolumePropertyRequest modifiedRequest = getOmRequest()
+    SetVolumePropertyRequest.Builder modifiedRequestBuilder = getOmRequest()
```
Comment (cxorm) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetOwnerRequest.java:68]: The variable name is a little confused IMHO.
Do you mind that we change it to setPropertyRequestBuilder ?

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java [hunk 2]
```diff
@@ -71,12 +71,12 @@ public OMVolumeSetQuotaRequest(OMRequest omRequest) {
   public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
 
     long modificationTime = Time.now();
-    SetVolumePropertyRequest modifiedRequest = getOmRequest()
+    SetVolumePropertyRequest.Builder modifiedRequestBuilde = getOmRequest()
```
Comment (cxorm) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java:74]: The same as above.

**Review by cxorm — COMMENTED** [review]
Thank you @aeioulisa for the work, overall looks good.
A minor comment is added inline.

**Review by cku328 — APPROVED** [review]

**Review by cxorm — APPROVED** [review]
Thanks @aeioulisa for updating it based on the comments.
I'm +1.

**Comment by cxorm** [comment]
Thanks @aeioulisa for the contribution and @cku328 for the review.
I have committed it.

**Pull request merged**